commit 255db24156cca78739d16bded6b4874bc860d124
Author: Mark Emlyn David Thomas <markt@apache.org>
Date:   Sun Jan 16 16:17:00 2011 +0000

    Add a new filter for fixes that require an API breaking change that need to wait until post 7.0.x
    
    git-svn-id: https://svn.apache.org/repos/asf/tomcat/trunk@1059600 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/res/findbugs/filter-post-7.0.x-fixes.xml b/res/findbugs/filter-post-7.0.x-fixes.xml
new file mode 100644
index 0000000..5a0b9a8
--- /dev/null
+++ b/res/findbugs/filter-post-7.0.x-fixes.xml
@@ -0,0 +1,41 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<FindBugsFilter>
+  <!-- Issues that should be fixed but can't be without API breaking changes -->
+  <!-- so wait until next release-->
+  <!-- Need to re-factor method names. protected do[Read|Write]Object? -->
+  <Match>
+    <Class name="org.apache.catalina.ha.session.DeltaSession" />
+    <Method name="readObject" />
+    <Bug code="Se" />
+  </Match>
+  <Match>
+    <Class name="org.apache.catalina.ha.session.DeltaSession" />
+    <Method name="writeObject" />
+    <Bug code="Se" />
+  </Match>
+  <Match>
+    <Class name="org.apache.catalina.session.StandardSession" />
+    <Method name="readObject" />
+    <Bug code="Se" />
+  </Match>
+  <Match>
+    <Class name="org.apache.catalina.session.StandardSession" />
+    <Method name="writeObject" />
+    <Bug code="Se" />
+  </Match>
+</FindBugsFilter>
\ No newline at end of file
